src/vivarium_gates_iv_iron/model_specifications/sub-saharan_africa.yaml

```diff
@@ -50,3 +51,7 @@ configuration:
             by_age: True
             by_sex: False
             by_year: True
+        maternal_disorders:
+            by_age: False
```

Comment:
Same here, we'll fix this in results processing. We'll want `by_age` and `by_year` to be `True`